pkg/minikube/node/start.go

```diff
@@ -471,8 +471,15 @@ func waitForCRIVersion(runner cruntime.CommandRunner, socket string, wait int, i
 
 	klog.Infof("Will wait %ds for crictl version", wait)
 
+	cmd := exec.Command("which", "crictl")
+	rr, err := runner.RunCmd(cmd)
+	if err != nil {
+		return err
+	}
+	crictl := strings.TrimSuffix(rr.Stdout.String(), "\n")
```

Comment:
maybe use  "TrimSpace" ? for removing all sort of unexpected spaces ?

Reply:
There is no whitespace expected, except for the newline from the `which` command

```console
$ which crictl | cat -vte
/usr/local/bin/crictl$
```